tendermint/tests/support/lite/single_step_sequential/commit_tests.json

```diff
@@ -0,0 +1,2360 @@
+ 							"next_validators_hash": "C403871B77A3295FFF1EE321424FC969EFAD87EF16740DF28A1B6CA62CA0977C",
+ 							"consensus_hash": "048091BC7DDC283F77BFBF91D73C44DA58C3DF8A9CBC867405D8B7F3DAADA22F",
+ 							"app_hash": "6170705F68617368",
+ 							"last_results_hash": "",
+ 							"evidence_hash": "",
+ 							"proposer_address": "026CC7B6F3E62F789DBECEC59766888B5464737D"
+ 						},
+ 						"commit": {
+ 							"block_id": {
+ 								"hash": "3F474672E168C53C793CA418F2A236B9CF94A0D66C3A87E6EF2E52FFD7E745BF",
+ 								"parts": {
+ 									"total": "1",
+ 									"hash": "A25578DC6C4E0E9A6E1BAACC40BE6F788C7464CE46F1A7713ED8DAE3744DF501"
+ 								}
+ 							},
+ 							"precommits": [
+ 								null,
+ 								{
+ 									"type": 2,
+ 									"height": "2",
+ 									"round": "1",
+ 									"block_id": {
+ 										"hash": "3F474672E168C53C793CA418F2A236B9CF94A0D66C3A87E6EF2E52FFD7E745BF",
+ 										"parts": {
+ 											"total": "1",
+ 											"hash": "A25578DC6C4E0E9A6E1BAACC40BE6F788C7464CE46F1A7713ED8DAE3744DF501"
+ 										}
+ 									},
+ 									"timestamp": "2019-11-02T15:04:15Z",
+ 									"validator_address": "026CC7B6F3E62F789DBECEC59766888B5464737D",
+ 									"validator_index": "1",
+ 									"signature": "WFrbDfUXQqkR+cxog0axwaNLB/qrGqduGI/O1ZcwxYzTejL5b4YkN4iEb2FQzHSBzOFVY56wDQmzRo6P06L1Cw=="
+ 								},
+ 								{
+ 									"type": 2,
+ 									"height": "2",
+ 									"round": "1",
+ 									"block_id": {
+ 										"hash": "3F474672E168C53C793CA418F2A236B9CF94A0D66C3A87E6EF2E52FFD7E745BF",
+ 										"parts": {
+ 											"total": "1",
+ 											"hash": "A25578DC6C4E0E9A6E1BAACC40BE6F788C7464CE46F1A7713ED8DAE3744DF501"
+ 										}
+ 									},
+ 									"timestamp": "2019-11-02T15:04:15Z",
+ 									"validator_address": "03A238BCAF7D1626DFE8A4AFB9448D00B7A3D2E2",
+ 									"validator_index": "2",
+ 									"signature": "DieRgoRSJz7/T41fIQ3UJ9XI+/fGYFypTbKIKnKhVTRzyMo7ybcIcdX6iptHOoFQL1YmM6b1nAhVz+qxfMO1Ag=="
+ 								}
+ 							]
+ 						}
+ 					},
+ 					"validator_set": {
+ 						"validators": [
+ 							{
+ 								"address": "01F527D77D3FFCC4FCFF2DDC2952EEA5414F2A33",
+ 								"pub_key": {
+ 									"type": "tendermint/PubKeyEd25519",
+ 									"value": "OAaNq3DX/15fGJP2MI6bujt1GRpvjwrqIevChirJsbc="
+ 								},
+ 								"voting_power": "50",
+ 								"proposer_priority": "-50"
+ 							},
+ 							{
+ 								"address": "026CC7B6F3E62F789DBECEC59766888B5464737D",
+ 								"pub_key": {
+ 									"type": "tendermint/PubKeyEd25519",
+ 									"value": "+vlsKpn6ojn+UoTZl+w+fxeqm6xvUfBokTcKfcG3au4="
+ 								},
+ 								"voting_power": "50",
+ 								"proposer_priority": "-50"
+ 							},
+ 							{
+ 								"address": "03A238BCAF7D1626DFE8A4AFB9448D00B7A3D2E2",
+ 								"pub_key": {
+ 									"type": "tendermint/PubKeyEd25519",
+ 									"value": "b6hwk3pjiOTJfLVCcLDA3I3lO71zWJ0VSded5LUl9T0="
+ 								},
+ 								"voting_power": "50",
```

Comment:
That's the next test that is failing. Quick question: Should we rename this to: "Case: one lite block, more than two-third vals did sign, no error". I was first confused why there is no error if less than 1/3 signed :-D 

This one seems to fail on the newly introduced check in regarding the parts.hash btw.

Reply:
Oh it's "less than one-third don't sign" which is same as "more than two-thirds did sign". That's why it does not error on the go side too.

Reply:
Me wondering was more about the parts hash. I understand the equiv. of both and suggested to rename the description to the latter "more than two-thirds did sign" as it is less confusing (but maybe it's just me).

Reply:
Ah okay. I can do that